solana/automata-dcap-framework/programs/automata-on-chain-pccs/src/state.rs

```diff
@@ -0,0 +1,172 @@
+use anchor_lang::prelude::*;
+
```

Comment:
1. Why is there an `Owner` field in all collateral states? There shouldn't be any sort of ownership check, any one can write to the PDA, as long as we can verify that the collateral data provided is signed by Intel.

2. We probably need to add a `Verified` boolean flag for all collaterals. We can then, add API methods (both onchain and ZK) to perform ECDSA verifications to toggle the `Verified` flag. (this can get tricky with Borsh serialized data)

Reply:
Please resolve (1), we shouldn't be needing an `Owner` field for collateral PDAs

Reply:
will be addressed in a separate PR